files-with-type

```diff
@@ -10,4 +10,4 @@
 mime_type=$1
```

Comment:
Not directly related to the PR but ... Docs: find is not used anymore. I would actually remove the dependency remark since it's easily going to bitrot.